source/help/policies/submission_agreement.md

```diff
@@ -1,56 +1,78 @@
 
-The Submitter may elect to make the Work available under one of the following Creative Commons licenses that the Submitter shall select at the time of submission:
-Creative Commons Attribution license (CC BY 4.0);
-Creative Commons Attribution-ShareAlike license (CC BY-SA 4.0);
-Creative Commons Attribution-Noncommercial-ShareAlike license (CC BY-NC-SA 4.0);
-Creative Commons Attribution-NonCommercial-NoDerivatives license (CC BY-NC-ND 4.0);
-arXiv license (https://arxiv.org/licenses/nonexclusive-distrib/1.0/license.html); or
-Creative Commons Public Domain Dedication (CC0 1.0).
+You may elect to make the Work available under one of the following licenses that you shall select at the time of submission: 
 
-If you wish to use a different CC license, then select arXiv's non-exclusive license to distribute in the arXiv submission process and indicate the desired Creative Commons license in the actual article.
+-  Creative Commons Attribution license (CC BY 4.0) (https://creativecommons.org/licenses/by/4.0/); 
+-  Creative Commons Attribution-ShareAlike license (CC BY-SA 4.0) (https://creativecommons.org/licenses/by-sa/4.0/); 
+-  Creative Commons Attribution-Noncommercial-ShareAlike license (CC BY-NC-SA 4.0) (https://creativecommons.org/licenses/by-nc-sa/4.0/);
+-  Creative Commons Attribution-NonCommercial-NoDerivatives license (CC BY-NC-ND 4.0) (https://creativecommons.org/licenses/by-nc-nd/4.0/);
+-  arXiv license (https://arxiv.org/licenses/nonexclusive-distrib/1.0/license.html); or 
+-  Creative Commons Public Domain Dedication (CC0 1.0) (https://creativecommons.org/share-your-work/public-domain/cc0/).
+    
+If you wish to use a different CC license, then select arXiv’s non-exclusive license to distribute in the arXiv submission process and indicate the desired Creative Commons license in the actual article. 
 
-The Creative Commons licenses are explained here:
-https://creativecommons.org/licenses/.
+The Creative Commons licenses are explained here: https://creativecommons.org/licenses/.
 
-### Metadata license
-
-To the extent that the Submitter or arXiv has a copyright interest in metadata accompanying the submission, a Creative Commons CC0 1.0 Universal Public Domain Dedication will apply. Metadata includes title, author, abstract, and other information describing the Work.
+You should take care to submit a work to arXiv _only if you are certain_ that you will not later wish to publish it in a journal or other outlet that prohibits prior distribution on an e-print server. _arXiv will not remove an announced article in order to comply with such a journal policy – the license granted upon submission is irrevocable._ 
```

Comment:
Can that sentence-level hyphen be changed to an em dash? :)

Reply:
I think the it already is, based on the preview. 
![Screen Shot 2023-03-15 at 8 45 00 AM](https://user-images.githubusercontent.com/4266002/225327559-1747b79a-ce35-47f6-9261-65f737da89f9.png)

Reply:
Yeah I'm not sure which it is either, now that I'm looking at it. I think in md it's just adding a `--` instead of `-`, though I suspect that it supports the convention of entering the actual utf8 character `—` .

Reply:
done